src/include/stat.h

```diff
@@ -170,25 +172,26 @@ __wt_stats_clear(void *stats_arg, int slot)
             (void)__wt_atomic_addi64(&(stat)->fld, (int64_t)(value)); \
     } while (0)
 
-#define WT_STAT_DECRV(session, stats, fld, value)                                 \
-    do {                                                                          \
-        WT_STAT_DECRV_BASE(session, (stats)[(session)->stat_bucket], fld, value); \
+#define WT_STAT_DSRC_DECRV(session, stats, fld, value)                                 \
+    do {                                                                               \
+        WT_STAT_DECRV_BASE(session, (stats)[(session)->stat_dsrc_bucket], fld, value); \
     } while (0)
-#define WT_STAT_DECRV_ATOMIC(session, stats, fld, value)                                 \
-    do {                                                                                 \
-        WT_STAT_DECRV_ATOMIC_BASE(session, (stats)[(session)->stat_bucket], fld, value); \
+#define WT_STAT_DSRC_DECRV_ATOMIC(session, stats, fld, value)                                 \
```

Comment:
AWT_STAT_DSRC_DECRV_ATOMIC and  WT_STAT_DSRC_INCRV_ATOMIC are unused, is there a reason we want to keep these or can I remove them?

Reply:
Probably could remove them (it's easy enough to add when/if we need them again).  I can see that there are connection wide stats that we want to be absolutely accurate, but session stats never seem that precise, off by a few is usually not a problem.